packages/gateway/scripts/cli.js

```diff
@@ -18,52 +25,178 @@ prog
+  const version = `${pkg.name}@${pkg.version}-${opts.env}+${git.short(
+    __dirname
+  )}`
+
+  await build({
+    entryPoints: [path.join(__dirname, '../src/index.js')],
+    bundle: true,
+    format: 'esm',
+    outfile: 'dist/index.mjs',
+    legalComments: 'external',
+    define: {
+      VERSION: JSON.stringify(version),
+      COMMITHASH: JSON.stringify(git.long(__dirname)),
+      BRANCH: JSON.stringify(git.branch(__dirname)),
+      ENV: opts.env || 'dev',
+      global: 'globalThis',
+    },
+    minify: opts.env === 'dev' ? false : true,
+    sourcemap: true,
+  })
+
+  // Sentry release and sourcemap upload
+  if (process.env.SENTRY_UPLOAD === 'true') {
+    const cli = new Sentry(undefined, {
+      authToken: process.env.SENTRY_TOKEN,
+      org: 'protocol-labs-it',
+      project: 'nft-gateway',
+      dist: git.short(__dirname),
     })
 
-    // Sentry release and sourcemap upload
-    if (process.env.SENTRY_UPLOAD === 'true') {
-      const cli = new Sentry(undefined, {
-        authToken: process.env.SENTRY_TOKEN,
-        org: 'protocol-labs-it',
-        project: 'nft-gateway',
-        dist: git.short(__dirname),
-      })
-
-      await cli.releases.new(version)
-      await cli.releases.setCommits(version, {
-        auto: true,
-        ignoreEmpty: true,
-        ignoreMissing: true,
-      })
-      await cli.releases.uploadSourceMaps(version, {
-        include: ['./dist'],
-        urlPrefix: '/',
-      })
-      await cli.releases.finalize(version)
-      await cli.releases.newDeploy(version, {
-        env: opts.env,
-      })
+    await cli.releases.new(version)
+    await cli.releases.setCommits(version, {
+      auto: true,
+      ignoreEmpty: true,
+      ignoreMissing: true,
+    })
+    await cli.releases.uploadSourceMaps(version, {
+      include: ['./dist'],
+      urlPrefix: '/',
+    })
+    await cli.releases.finalize(version)
+    await cli.releases.newDeploy(version, {
+      env: opts.env,
+    })
+  }
+}
+
+async function ipfsCmd({ start, stop }) {
+  const project = 'ipfs-daemon'
+  const composePath = path.join(__dirname, '../docker/docker-compose.yml')
+
+  if (start) {
+    if (await isPortReachable(8080)) {
+      throw new Error(
+        'IPFS daemon is already running. Please check if you have any docker project or cluster deamon already running.'
+      )
     }
+
+    await execa('docker-compose', [
+      '--file',
+      composePath,
+      '--project-name',
+      project,
+      'up',
+      '--detach',
+    ])
+
+    await pWaitFor(async () => {
+      const { stdout } = await execa('docker', ['logs', '-t', 'ipfs0'])
+      return stdout.includes('Daemon is ready')
+    })
+    console.log('docker started')
+
+    // Add fixture files
+    const client = create({ url: IPFS_API_URL })
+
+    await Promise.all([
+      // bafkreidyeivj7adnnac6ljvzj2e3rd5xdw3revw4da7mx2ckrstapoupoq
+      client.add('Hello nft.storage! 😎', {
+        chunker: 'size-1048576',
+        rawLeaves: true,
```

Comment:
We don't override these settings in NFT.Storage - I think this might be confusing.